dsci522_environment.yml

```
@@ -0,0 +1,10 @@
+name: dsci522
```

We could opt for a more descriptive name
```suggestion
name: dsci522-airline-pred
```